src/main/java/frc/robot/subsystems/Drivetrain/DriveImpl.java

```diff
@@ -23,13 +22,13 @@
 public class DriveImpl extends RepeatingPooledSubsystem implements Drive {
 
     // motor controller groups
-    private CANSparkMax leftSide;
-    private CANSparkMax rightSide;
+    public CANSparkMax leftSide;
+    public CANSparkMax rightSide;
```

Comment:
I don't think these need to be public, do they? I can't see why this change was made.

Reply:
The left and right motor controllers need to be accessed by the Autonomous.java file, can't do that if they are private. I only need them to get the integrated pid controllers for each side. The only work around I could think of was using the WPI pidcontroller which would have taken a bit extra math I believe. I just thought changing accessibility would be easier.

Reply:
I don't think that's how we want to go about it. The Drivetrain/DrivetrainImpl should have methods that the Autonomous can call, and let the DrivetrainImpl deal with how to actually use the motors. Don't directly expose the motors for other subsystems or other logic to interfere with or else it quickly becomes very tough to troubleshoot problems when you aren't sure where things are actually running and if things are interfering with each other.